src/charm.py

```diff
@@ -366,12 +469,20 @@ def _clear_all_configs_and_restart_traefik(self):
         # configuration files on a storage volume that survives the pod churn, before
         # we start traefik we clean up all Juju-generated config files to avoid spurious
         # routes.
+        self._clear_tracing_config()
```

Comment:
Should we instead move all config to non-persistent storage?

Reply:
also see: https://github.com/canonical/traefik-k8s-operator/pull/213